app/nexus/internal/initialization/recovery/recovery.go

```diff
@@ -340,87 +307,3 @@ func NewPilotRecoveryShards() map[int]*[32]byte {
 		"message", "Successfully generated pilot recovery shards.")
 	return result
 }
-
-// BootstrapBackingStoreWithNewRootKey initializes the backing store with a new
-// root key if it hasn't been bootstrapped already. It generates a new AES-256
-// root key, initializes the state with this key, and distributes key shards
-// to all configured keepers.
-//
-// The function requires the number of keepers to match the configured Shamir
-// shares. It continuously attempts to distribute shards to all keepers until
-// successful, waiting 5 seconds between retry attempts. The backing store is
-// initialized before keeper distribution to allow immediate operation.
-//
-// Parameters:
-//   - source *workloadapi.X509Source: An X509Source used for authenticating
-//     with keeper nodes
-//
-// The function will crash fatally if:
-//   - Root key creation fails
-//   - The number of keepers doesn't match the configured Shamir shares
-func BootstrapBackingStoreWithNewRootKey(source *workloadapi.X509Source) {
```

Comment:
Not using this anymore since we don't bootstrap SPIKE Nexus from within SPIKE Nexus anymore.